src/chrome/content/rules/FreeBSoft.org.xml

```diff
@@ -23,8 +21,11 @@
 -->
 <ruleset name="FreeBSoft.org (partial)">
 
-	<target host="its.freebsoft.org" />
-	<target host="lists.freebsoft.org" />
+	<target host=".freebsoft.org" />
```

Comment:
Errors with validation. Add wildcard attribute if that was what you meant to do. `*.freebsoft.org`